plugins/registry.js

```diff
@@ -125,7 +125,7 @@ export default class PluginRegistry {
             icon: resolveReactElement(icon),
             action,
             dropdownText: resolveReactElement(dropdownText),
-            tooltipText,
+            tooltipText: resolveReactElement(tooltipText),
```

Comment:
Can we update the comment to state it takes a react element as well?

Reply:
Mais oui!